prpr/locales/fr-FR/ending.ftl

```diff
@@ -1,17 +1,17 @@
 
 retry = RECOMMENCER
-proceed = PROCÉDER
+proceed = CONTINUER
 
-new-best = NOUVEAU MEILLEUR
-max-combo = COMBO MAXIMUM
+new-best = NOUVEAU RECORD
+max-combo = COMBO MAX
```

Comment:
Shortened this, because the text overflows with the next element
![image](https://github.com/user-attachments/assets/036099ff-f5a3-4205-98cf-1fc264601a79)